### PR TITLE
St petersburg collection

### PR DESCRIPTION
**Pull request by DenisNosnitsin1970**
Please check.

### CODE REVIEW

File: StPetersburg/RNBefns7.xml [hunk 1]
```diff
@@ -24,136 +24,139 @@ schematypens="http://relaxng.org/ns/structure/1.0"
...
                            <dimensions type="outer" unit="mm">
                               <height>18</height>
                               <width>16</width>
                               <depth/>
                            </dimensions>
-                           <note>the indication of size in <bibl> 
-                                 <ptr target="bm:Turaev1906rukopisi"/>
-                                 <citedRange unit="page">22</citedRange> 0,18x0,16 cm must be a mistake </bibl>
+                           <note>The indication of size in <bibl>
+                                 <ptr target="bm:Turaev1906rukopisi"/><citedRange unit="page">22</citedRange> as
```
Comment (PietroLiuzzo) [inline StPetersburg/RNBefns7.xml]: please remove text from element bibl

File: StPetersburg/RNBefns7.xml [hunk 2]
```diff
@@ -24,136 +24,139 @@ schematypens="http://relaxng.org/ns/structure/1.0"
...
+                           <note>Four initial leaves are unnumbered blank paper leaves (here numbered i-iv).</note>
                         </extent>
-                
+
                         <foliation>
-                           The leaves of the first paper quire (i-iv) have no numbers. Pagination on verso-sides of the folios in <foreign xml:lang="Coptic"/>, from  112 to 195 </foliation>
-                        <collation> The first four unnombered leaves (to be counted as i-iv) are made of paper </collation>
-                
+                           The leaves of the first paper quire (i-iv) have no numbers. Pagination on verso-sides of the folios
+                            in <foreign xml:lang="Coptic"/>, from  112 to 195.</foliation>
```
Comment (PietroLiuzzo) [inline StPetersburg/RNBefns7.xml]: I think the element foreign should contain the coptic text, not be used as empty element before that text, if the squares here are the coptic

File: StPetersburg/RNBefns7.xml [hunk 3]
```diff
@@ -24,136 +24,139 @@ schematypens="http://relaxng.org/ns/structure/1.0"
...
                      <binding contemporary="false">
-                        <decoNote xml:id="b1"> Binding not Ethiopian, made of tooled Morocco leather. 
-                  One pair of sewing stations can be assumed (not indicated in the catalogue)</decoNote> 
-                        <decoNote xml:id="b2" type="bindingMaterial">
-                           <material key="wood"/>
-                        </decoNote>
-                        <decoNote xml:id="b3" type="bindingMaterial">
-                           <material key="leather"/>
+                        <decoNote xml:id="b1">Binding non-Ethiopian, made of tooled Morocco leather.</decoNote>
+                <decoNote xml:id="b2">Probably one pair of sewing stations.</decoNote>
```
Comment (PietroLiuzzo) [inline StPetersburg/RNBefns7.xml]: where is b3?

File: StPetersburg/RNBefns7.xml [hunk 4]
```diff
@@ -186,61 +193,40 @@ schematypens="http://relaxng.org/ns/structure/1.0"
...
       <revisionDesc>
          <change who="DN" when="2016-12-12">Created catalogue entry</change>
          <change who="DN" when="2016-12-21">Additions</change>
       </revisionDesc>
   </teiHeader>
-  
+
   <text xml:base="https://betamasaheft.eu/">
       <body>
-         <div type="edition">
```
Comment (PietroLiuzzo) [inline StPetersburg/RNBefns7.xml]: why was this text removed?

Comment (DenisNosnitsin1970) [reply StPetersburg/RNBefns7.xml]: It was incipit and explicit.

File: StPetersburg/RNBefns7.xml (hunk 3 above)
Comment (DenisNosnitsin1970) [reply StPetersburg/RNBefns7.xml]: introduced

Comment (PietroLiuzzo) [reply StPetersburg/RNBefns7.xml]: please make a further commit with this updates to this branch

File: StPetersburg/RNBefns7.xml (hunk 1 above)
Comment (DenisNosnitsin1970) [reply StPetersburg/RNBefns7.xml]: ok

File: StPetersburg/RNBefns7.xml (hunk 2 above)
Comment (DenisNosnitsin1970) [reply StPetersburg/RNBefns7.xml]: ok. actually i do not see Coptic numbers any longer.

**Pull request merged**